config/tailwind.js

```diff
@@ -205,7 +205,7 @@ module.exports = {
       'Fira Sans',
       'Droid Sans',
       'Helvetica Neue',
-      'sans-serif',
+      'sans-serif'
```

Comment:
prefer to leave the trailing comma here so that if new items are added to the array, the diff is only one line